--- actions.original.py	2011-09-15 10:39:05.000000000 +0400
+++ actions.py	2011-09-15 00:10:44.000000000 +0400
@@ -332,10 +332,11 @@
     # unless it is triggered from a menu
     # or if it is not tied to an object (ex: home action after login)
     open_new_tab = False
-    if action['target'] == 'current' and action.get('res_model') != 'ir.ui.menu'\
-        and data.get('model') != 'ir.ui.menu' and not 'home_action' in data:
-        action['target'] = 'popup'
-        open_new_tab = True
+#DSK: fucking popups will not brake our life!
+#    if action['target'] == 'current' and action.get('res_model') != 'ir.ui.menu'\
+#        and data.get('model') != 'ir.ui.menu' and not 'home_action' in data:
+#        action['target'] = 'popup'
+#        open_new_tab = True
 
     # search_view key in action is >8k added to the URL every time, which
     # breaks firefox (and probably Apache) as it's shoved into a header and
